src/pretix/helpers/thumb.py

```diff
@@ -171,24 +171,35 @@ def create_thumbnail(sourcename, size):
     except:
         raise ThumbnailError('Could not load image')
 
-    image = resize_image(image, size)
+    frames = [resize_image(frame, size) for frame in ImageSequence.Iterator(image)]
+    image_out = frames[0]
+    save_kwargs = {}
 
     if source.name.lower().endswith('.jpg') or source.name.lower().endswith('.jpeg'):
         # Yields better file sizes for photos
         target_ext = 'jpeg'
         quality = 95
+    elif source.name.lower().endswith('.gif'):
+        target_ext = 'gif'
```

Comment:
What about animated png? Browser-support is quite good and PIL supports it: https://caniuse.com/?search=apng

```suggestion
    elif source.name.lower().endswith('.gif') or source.name.lower().endswith('.png'):
        target_ext = source.name.lower()[-3:]
```